linux/linux.mk

```diff
@@ -52,4 +53,4 @@ INCLUDES += -I$(LIBOBS_INCLUDES)
 INCLUDES += -I$(FFMPEG_INCLUDES)
 
 LDD_LIBS += -lobs
-LDD_FLAG += -shared
+LDD_FLAG += $(LDFLAGS) -shared
```

Comment:
Is LDFLAGS some sort of builtin ? Where does it come from ?

Reply:
Sorry I disappeared 3 weeks ago.  So I did a bit more investigation on $(LDFLAGS) and I even did a build echoing the value of that variable and saw that it was blank.  This was left over from a patch from the previous maintainer for the AUR package.  I've now removed it.  Sorry for the confusion.